core: Add USW2-RDS:ChargedBackupUsage to types list

pwithnall opened the pull request:
Signed-off-by: Philip Withnall <pwithnall@endlessos.org>

## Description of Change

Adds a missing suffix check for `RDS:ChargedBackupUsage` to set it as using a HDD. This fixes the warning:
```
api_1     | 2021-05-25T10:03:01.861Z [RDS Storage Logger] warn: Unexpected Cost explorer Dimension Name: USW2-RDS:ChargedBackupUsage 
```

## Checklist

- [x] PR description included and stakeholders cc'd

dtoakley commented on packages/core/src/services/aws/CostAndUsageTypes.ts:
```diff
@@ -53,6 +53,7 @@ export const HDD_USAGE_TYPES: string[] = [
   'GlacierByteHrs', // Glacier
   'Aurora:BackupUsage', // Aurora back up in S3
   'RDS:ChargedBackupUsage', // RDS Backup in S3
+  'USW2-RDS:ChargedBackupUsage', // RDS Backup in S3
```
Hi @pwithnall -- thanks for the PR and apologies for the delayed response. This usage type should be handled by the string above it: `RDS:ChargedBackupUsage`. This is because in the CostAndUsageReports class we check that the string ends with any of the options in this array: `this.endsWithAny(HDD_USAGE_TYPES, usageType)`

Let me know if I'm missing something though and it's not being handled for you!

pwithnall replied:
I put this PR together because the server was printing a warning about `USW2-RDS:ChargedBackupUsage` not being handled, so something’s amiss. I followed the instructions in the documentation about handling those warnings to put this PR together. However, I haven’t tested it, so it could be the wrong fix.

dtoakley replied:
Hi @pwithnall - would you be able to pull the latest from the trunk branch and re-test and see if the warning is still showing? If it is, could you also log the entire `costAndUsageReportRow`, but changing line 212 of the `CostAndUsageReport` class:

`this.costAndUsageReportsLogger.warn(`Unexpected usage type for storage service: ${costAndUsageReportRow}`)`

pwithnall replied:
OK, I’ll give that a go when I get a chance in the next week or two, thanks.

pwithnall replied:
I dug into it a bit more and actually chased down the warning message, and it’s not a `CostAndUsageTypes` thing at all. I’ve updated the PR with a tested fix.

pwithnall committed:
aws: Add RDS:ChargedBackupUsage as a HDD storage type

This fixes the warning:
```
[RDS Storage Logger] warn: Unexpected Cost explorer Dimension Name: USW2-RDS:ChargedBackupUsage
```

Signed-off-by: Philip Withnall <pwithnall@endlessos.org>

dtoakley commented:
@pwithnall -- I made a small tweak so it's consistent with the other storage types being used, and add a test case for this usage type, and merge it in with this PR: https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/pull/303

Going to close this PR and it's all looking good. Thanks so much @pwithnall -- this is a great improvement!
